### ISSUE TITLE
HDR to SDR

### ISSUE THREAD

**Issue by Marciuz90** [RandomEngy/VidCoder#797]
Hi, what settings do you need to make an HDR to SDR?

**Comment by RandomEngy** [comment]
Set Max width and Max height on the Sizing tab.

**Comment by Marciuz90** [comment]
What?

Ottieni Outlook per Android<https://aka.ms/ghei36>

________________________________

**Comment by RandomEngy** [comment]
You could set your interface to English then it might be easier to find those settings.

**Comment by Marciuz90** [comment]
no, I meant how not to consider the master-Display g, the bt2020 and the cll, in order to have a bt709 and a cll equal to 0? maybe you can create a public ticket and hear who uses your program, so they can help me with it. Thank you

**Comment by RandomEngy** [comment]
Don't know anything about color space conversions. You might ask the HandBrake team about that.

**Comment by Marciuz90** [comment]
ok
